app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -20,24 +20,28 @@ def self.include_long_dredging?
       includes(:exemption).where(flood_risk_engine_exemptions: { code: Exemption::LONG_DREDGING_CODES }).any?
     end
 
-    def latest_decision
+    def accept_reject_decision_user
+      User.find_by_id(accept_reject_decision_user_id).try(:email)
```

Comment:
Is it worth setting this up as an association?

`belongs_to :accept_reject_decision_user, class_name: 'User'` or something like that.

Then this method would be called `accept_reject_decision_user_email`